### DISCUSSION TITLE
broke chrome, ubuntu, docker desktop and chrome

### DISCUSSION THREAD

**Discussion by 1mike12** [teeotsa/windows-11-debloat#23] [General]
Just an fyi I wish I had before I ran this. I took a peek at the issues and script source and thought, hey, should be fine, there's a restore point created. 

Well, my experience has been that even with the restore point, I cannot fix the fact that chrome has broken along with ubuntu. So now I have to spend time getting those to work again.

**Comment by teeotsa** [comment]
What's wrong with Chrome & Ubuntu?
